cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -53,3 +55,42 @@ def create_sketch(ctx, name, description):
         description = name
     sketch = api_client.create_sketch(name=name, description=description)
     click.echo(f"Sketch created: {sketch.name}")
+
+
+@sketch_group.command("add_event")
+@click.option("--message", required=True, help="Message of the event.")
+@click.option(
+    "--date",
+    required=True,
+    help="Date of the event (ISO 8601). Example: 2023-03-08T10:59:24+00:00",
+)
+@click.option(
+    "--attributes",
+    required=False,
+    help="Attributes of the event. Example: key1=value1,key2=value2",
+)
+@click.option(
+    "--timestamp_desc",
+    required=False,
+    help="Timestamp description of the event.",
+)
+@click.pass_context
+def add_event(ctx, message, date, attributes, timestamp_desc="Timecketh-cli"):
+    """Add an event to the sketch."""
+    sketch = ctx.obj.sketch
+
+    if attributes:
+        attributes = attributes.split(",")
+        attributes = [attribute.split("=") for attribute in attributes]
+        attributes = {
+            key: value for key, value in attributes
+        }  # pylint: disable=unnecessary-comprehension
```

Comment:
Risky to call all of these with the same variable name. Prefer `attribute_str`, `attribute_pairs`, `attribute_tuples`

Reply:
should be better now, wdyt?